modules/standard/IO.chpl

```diff
@@ -3698,6 +3845,59 @@ inline proc channel.read(ref args ...?k,
   return !error;
 }
 
+proc stringify(args ...?k):string {
```

Comment:
Actually I think I'll write user documentation for it. It's more or less the equivalent of what string.write used to do. I'm open to suggestions about whether or not it should be user facing and if it should get a better name. It just converts all the arguments to strings and then returns the result of concatenating them.